src/ProgressBar.php

```diff
@@ -118,56 +162,61 @@ public function size(string $value): self
     }
 
     /**
-     * Returns a new instance with the specified progress bar color.
+     * Returns a new instance with the specified value of the progress.
      *
-     * @param string $value The progress bar color.
+     * @param float|null $value The value of the progress. Set `0` to display loading animation.
      *
      * @return self
      */
-    public function color(string $value): self
+    public function value(?float $value): self
     {
-        if (!in_array($value, self::COLOR_ALL, true)) {
-            $values = implode('"', self::COLOR_ALL);
-            throw new InvalidArgumentException("Invalid color. Valid values are: \"$values\".");
+        if ($value < 0 || $value > 100) {
```

Comment:
Seems this condition edge case isn't covered by test.

Reply:
https://github.com/terabytesoftw/yii-bulma/blob/959aa9e66629fe59094f330678aa74a38387a34f/tests/ProgressBarTest.php#L90